pages.pl/windows/set.md

```diff
@@ -0,0 +1,20 @@
+# set
+
+> Wyświetl lub ustaw zmienne środowiskowe dla bieżącej instancji CMD.
+> Więcej informacji: <https://learn.microsoft.com/windows-server/administration/windows-commands/set>.
+
+- Lista wszystkich bieżących zmiennych środowiskowych:
+
+`set`
+
+- Ustaw zmienną środowiskową na określoną wartość:
+
+`set {{name}}={{value}}`
+
+- Wymień zmienne środowiskowe zaczynające się od podanego ciągu znaków:
+
+`set {{name}}`
+
+- Pyta użytkownika o wartość dla określonej zmiennej:
+
+`set /p {{name}}={{prompt_string}}`
```

Comment:
Also here, can you translate the 2 arguments?

Reply:
The values have been updated